src/components/CategoriesContainer.js

```diff
@@ -1,11 +1,14 @@
 import React, { useState } from 'react';
-import store from '../redux/configureStore';
+import { useDispatch, useSelector } from 'react-redux';
+import { checkStatus } from '../redux/categories/categories';
 
 const CategoriesContainer = () => {
   const [dynamicContent, setDynamicContent] = useState([]);
-  const categories = store.getState().categories[0];
+  const dispatch = useDispatch();
+  const status = useSelector((state) => state.categories);
```

Comment:
- [ ] Please make sure that when the user clicks the `Check status` button, the message `"Under construction"` is displayed. At the moment, I have to click twice to get this message. Kindly look into it and fix it! 🤓 🔧